google/cloud/_helpers.py

```diff
@@ -455,7 +455,7 @@ def _datetime_to_rfc3339(value, ignore_zone=True):
     return value.strftime(_RFC3339_MICROS)
 
 
-def _to_bytes(value, encoding='ascii'):
+def _to_bytes(value, encoding='utf-8'):
```

Comment:
Leave `ascii` as the default. Just pass in `utf-8` when using `_to_bytes` in the code that actually needs changed.